### DISCUSSION TITLE
How do you generate an image for loot drops?

### DISCUSSION THREAD

**Discussion by Kape98** [oldschoolgg/oldschoolbot#2206] [Q&A] [answered]
Hello
I wanted to make a bot of my own for a different game and wanted to ask how you did that. I'm kinda new to github and not very experienced with coding, but I got my bot working and I want it to send something similar to what your bot does with the +kill command.

**Comment by gc** [comment, marked as answer]
If you're using javascript, you need to use Canvas to generate images

https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API
https://github.com/Automattic/node-canvas

**Reply by Kape98** [reply]
Thanks!
